Podcast/SearchITunesTableViewController.swift

```diff
@@ -0,0 +1,123 @@
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+import NVActivityIndicatorView
+
+class SearchITunesTableViewController: ViewController, UITableViewDelegate, UITableViewDataSource, UISearchControllerDelegate, SearchSeriesTableViewDelegate, UISearchResultsUpdating, UISearchBarDelegate {
+
+    var searchResults: [Series] = []
+    let seriesCellIdentifier = "SeriesCell"
+    let seriesCellHeight: CGFloat = 95
+    
+    var loadingIndicatorView: NVActivityIndicatorView?
+    var tableView: EmptyStateTableView = EmptyStateTableView(frame: .zero, type: .search)
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        tableView.frame = view.frame
+        tableView.showsVerticalScrollIndicator = false
+        tableView.separatorStyle = .none
+        tableView.delegate = self
+        tableView.dataSource = self
+        view.addSubview(tableView)
+        mainScrollView = tableView
+        
+        loadingIndicatorView = createLoadingAnimationView()
+        view.addSubview(loadingIndicatorView!)
+        loadingIndicatorView?.snp.makeConstraints { make in
+            make.center.equalToSuperview()
+        }
+        
+    }
+    
+    func fetchData(query: String, completion: (() -> ())?)  {
+        System.endpointRequestQueue.cancelAllEndpointRequestsOfType(type: SearchEpisodesEndpointRequest.self)
+        System.endpointRequestQueue.cancelAllEndpointRequestsOfType(type: SearchUsersEndpointRequest.self)
+        System.endpointRequestQueue.cancelAllEndpointRequestsOfType(type: SearchSeriesEndpointRequest.self)
+        
+        tableView.backgroundView?.isHidden = true
+        loadingIndicatorView?.startAnimating()
+        
+        let request = SearchITunesEndpointRequest(query: query, offset: 0, max: 20)
+        request.success = { request in
+            guard let results = request.processedResponseValue as? [Series] else { return }
+            if results.isEmpty {
+                self.tableView.backgroundView?.isHidden = false
+            }
+            self.searchResults = results
+            self.tableView.reloadData()
+            self.loadingIndicatorView?.stopAnimating()
+        }
+        request.failure = { request in
+            self.tableView.backgroundView?.isHidden = false
+            self.loadingIndicatorView?.stopAnimating()
+        }
+        System.endpointRequestQueue.addOperation(request)
+        completion?()
+    }
+    
+    // MARK: TableView
+    
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return 1
+    }
+    
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        return seriesCellHeight
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        if let cell = tableView.dequeueReusableCell(withIdentifier: seriesCellIdentifier) as? SearchSeriesTableViewCell {
```

Comment:
if let cell = ....  as? SearchSeriesTableViewCell else { cell = SearchSeriesTableViewCell() }

then remove the duplicated code